openedx/core/djangoapps/user_api/verification_api/views.py

```diff
@@ -0,0 +1,30 @@
+""" Verification API v1 views. """
+from django.http import Http404
+from edx_rest_framework_extensions.authentication import JwtAuthentication
+from rest_framework.authentication import SessionAuthentication
+from rest_framework.generics import RetrieveAPIView
+from rest_framework.permissions import IsAuthenticated
+from rest_framework_oauth.authentication import OAuth2Authentication
+
+from lms.djangoapps.verify_student.models import SoftwareSecurePhotoVerification
+from openedx.core.djangoapps.user_api.verification_api.serializers import SoftwareSecurePhotoVerificationSerializer
+from openedx.core.lib.api.permissions import IsStaffOrOwner
+
+
+class PhotoVerificationStatusView(RetrieveAPIView):
+    """ PhotoVerificationStatus detail endpoint. """
+    authentication_classes = (JwtAuthentication, OAuth2Authentication, SessionAuthentication,)
+    permission_classes = (IsAuthenticated, IsStaffOrOwner,)
+    serializer_class = SoftwareSecurePhotoVerificationSerializer
+
+    def get_object(self):
+        username = self.kwargs['username']
+        verifications = SoftwareSecurePhotoVerification.objects.filter(user__username=username) \
+            .order_by('-updated_at').select_related('user')
```

Comment:
Once you remove the username field from the serializer, you won't need the call to `select_related`.